src/newcomponents/search/elements/SearchedUser.js

```diff
@@ -0,0 +1,45 @@
+import getUserFullTime from "@utils/user/get_user_full_name";
+import getUserInitials from "@utils/user/get_user_initials";
+import { addUsers } from "@store/Participants";
+import { insertChat } from "@store/Conversations";
+import { setSelectedConversation } from "@store/SelectedConversation";
+import { useDispatch } from "react-redux";
+import { useNavigate } from "react-router-dom";
+
+import "@newstyles/search/elements/SearchedUser.css";
+import api from "@api/api";
+
+export default function SearchedUser({ uObject }) {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
+  const createChat = async (uId) => {
+    if (!uId) {
+      return;
+    }
+
+    const requestData = {
+      type: "u",
+      participants: [uId],
+    };
+
+    const chat = await api.conversationCreate(requestData);
+    const users = await api.getParticipantsByCids({ cids: [chat._id] });
```

Comment:
do we really need to get participants from backend if we already have current user + uObject ?